mappings/net/minecraft/inventory/Inventory.mapping

```diff
@@ -11,7 +11,7 @@ CLASS agz net/minecraft/inventory/Inventory
 		ARG 2 stack
 	METHOD a canPlayerUseInv (Lawb;)Z
 		ARG 1 player
-	METHOD a getInvAmountOf (Lbbz;)I
+	METHOD a countInInv (Lbbz;)I
```

Comment:
This sounds weird.

Reply:
It counts the occurrence of an item type in this inventory. What do you suggest?

Reply:
this mapping seems perfectly fine to me

Reply:
getCountOf would make more sense if we ever remove the Inv parts of the method names for Inventory methods, but I think this change is fine.

Reply:
Same, it's fine to me